packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.types.ts

```diff
@@ -29,6 +29,9 @@ export interface ValuedReferenceAutocompleteProps {
   onDrop: (event: React.DragEvent<HTMLDivElement>) => void;
   onMoreClick: (event: React.MouseEvent<Element, MouseEvent>) => void;
   onCreateClick: (event: React.MouseEvent<Element, MouseEvent>) => void;
-  editReference: MutationFunction<GQLEditReferenceData, GQLEditReferenceVariables>;
   optionClickHandler: (element: GQLReferenceValue) => void;
+  clearReference: MutationFunction<GQLClearReferenceMutationData, GQLClearReferenceMutationVariables>;
```

Comment:
That's a very tight coupling with Apollo here, that's a bit odd

Reply:
Changed